NUnitTests/Lessons/21/EnumHomework.cs

```diff
@@ -1,42 +1,41 @@
 ﻿namespace Lesson21;
 
-// TODO: modify enum so CheckCustomIntNumbersForTestDataAgeEnum pass
 public enum TestDataAge
 {
-    Child,
-    Teenager,
-    Adult
+    Child = 7,
+    Teenager = 14,
+    Adult = 30
 }
 
-/* TODO: uncomment and implement tests so all Assert pass. Use such LINQ as Any(), Count(), Contains()
 [TestFixture]
 public class EnumHomework
 {
     [Test]
     public void CheckCustomIntNumbersForTestDataAgeEnum()
     {
 
-        Assert.That((int)TestDataAge.Child, Is.EqualTo(7));
-        Assert.That((int)TestDataAge.Teenager, Is.EqualTo(14));
-        Assert.That((int)TestDataAge.Adult, Is.EqualTo(30));
+        Assert.Multiple(() =>
+        {
+            Assert.That((int)TestDataAge.Child, Is.EqualTo(7));
+            Assert.That((int)TestDataAge.Teenager, Is.EqualTo(14));
+            Assert.That((int)TestDataAge.Adult, Is.EqualTo(30));
+        });
     }
 
     [Test]
     public void SomeIntCorrespondsToSomeTestDataAgeValue()
     {
         var listOfInt = new List<int>() { 5, 14, 15 };
-
-        var isAnyIntCorrespondsToTestDataAge = 
-
+        var isAnyIntCorrespondsToTestDataAge = listOfInt.Any(x => x == (int)TestDataAge.Teenager);
```

Comment:
we are interested to know if some int from listOfInt is equal to some int from TestDataAge. "some" means any of  Child,     Teenager, Adult, but not only to Teenager

Reply:
fixed please check

Reply:
thanks, closed